include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -122,29 +121,21 @@ namespace internal
                                         Number>;
 
     static void
-    evaluate(const unsigned int                            n_components,
-             const EvaluationFlags::EvaluationFlags        evaluation_flag,
-             const MatrixFreeFunctions::ShapeInfo<Number> &shape_info,
-             const Number *                                values_dofs_actual,
-             Number *                                      values_quad,
-             Number *                                      gradients_quad,
-             Number *                                      hessians_quad,
-             Number *                                      scratch_data);
+    evaluate(const unsigned int                     n_components,
+             const EvaluationFlags::EvaluationFlags evaluation_flag,
+             const Number *                         values_dofs_actual,
+             FEEvaluationData<dim, Number, false> & eval);
```

Comment:
Variable name `eval` is not intuitive in my opinion, given that the name `data` is used somewhere else for `MatrixFree` (why not `fe_eval_data`?).

Reply:
I agree, especially since `eval` is also used for something else in some of the functions, see https://github.com/dealii/dealii/blob/58c0c31147053395d69415a9fd2f8bb2735c38e6/include/deal.II/matrix_free/evaluation_kernels.h#L814-L818 
However, I do get pretty ugly indentation with `fe_eval_data`, so I went with the ubiquitous `fe_eval` instead (the problem is that the 80 character limit is pretty tough with the various namespaces and indentations here).